app/components/global/layout/DropdownNavLayout.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+import NavLink from '../../ui/atoms/NavLink'
+import Select, { createFilter, components } from 'react-select'
+
+const Root = styled.div``
+
+const DropdownNavLink = styled(NavLink)`
```

Comment:
I know this PR is still a WIP but I would suggest that we leave this part (i.e. _which_ content is going to get inserted into the dropdown menu) to another PR, which addresses #820 ?